docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+#
```

Comment:
I assume that the docker image this builds will be used as the build image for this repo? Very cool!

Reply:
Yep. It is the build image for this repo that we are using in the pipeline build. I was debating naming it Dockerfile.build to be more specific. 😄